src/components/Checkbox/checkbox.tsx

```diff
@@ -1,26 +1,31 @@
 "use client";
 
-import React, { useState } from "react";
-import Style from "./Checkbox.module.css";
+import { ChangeEvent, ComponentProps } from "react";
+import styles from "./Index.module.css";
 
-interface CheckboxProps {
-  checked: boolean;
-  onChange: () => void;
+interface ICheckboxProps extends ComponentProps<"input"> {
+  onCheckedChange?: (isChecked: boolean) => void;
```

Comment:
This `onCheckedChange` is not necessary because you're already in control of the `isChecked` property in the TodoItem component